addons/udes_stock/README.md

```diff
@@ -237,6 +242,34 @@ Additional Details:
 | get_picking_types | Returns a recordset with the picking_types of the warehouse. |
 
 
+## Addtional Information
+
+### Picking Naming Convention
+
+If a backorder gets created from a picking, then it will either have `-001` appended if the prefix does not exist, else it will causes it to increment one. How sequencing works is that the suffixed sequences refer to the level in the picking tree, not the nth backorder.
+
+Example:
+
+Ref        | Name       | Orignal | Parent | 1st Gen Children |
+-|-|-|-|-|
+Pick       | Pick001    | Pick001 |   -    | Pick1, Pick2     |
+Pick1      | Pick001-01 | Pick001 | Pick   | Pick3            |
+Pick2      | Pick001-01 | Pick001 | Pick   | Pick4, Pick5     |
+Pick3      | Pick001-02 | Pick001 | Pick1  |        -         |
+Pick4      | Pick001-02 | Pick001 | Pick2  |        -         |
+Pick5      | Pick001-02 | Pick001 | Pick2  |        -         |
+
+                Original Picking (Pick001)
```

Comment:
nit: can we use mermaid to display this?
```mermaid
  graph TD;
      A["Original Picking (Pick001)"] --> B["pick1 (Pick001-001)"];
      A --> C["pick 2  (Pick001-001)"];
      B --> D["pick3  (Pick001-002)"];
      C --> E["pick4  (Pick001-002)"];
      C --> F["pick5  (Pick001-002)"];
```

Reply:
note to do it you just do a code block like below and set formatting to `mermaid`:
```
  graph TD;
      A["Original Picking (Pick001)"] --> B["pick1 (Pick001-001)"];
      A --> C["pick 2  (Pick001-001)"];
      B --> D["pick3  (Pick001-002)"];
      C --> E["pick4  (Pick001-002)"];
      C --> F["pick5  (Pick001-002)"];
```